netlify/next-runtime
# update README image with new logo

## Description


Removes the old version of the README image and replaces it with one donning the new Netlify logo ✨

### Documentation



## Tests



1. Visit the [README](https://github.com/netlify/next-runtime/blob/kma/update-readme-img/README.md) file
2. See the change (updated Netlify logo)

## Relevant links (GitHub issues, etc.) or a picture of cute animal


![cute animal](https://res.cloudinary.com/cutelify/image/upload/c_fill,g_auto,q_auto:good,w_300,h_350/noah-silliman-BzHNKFUGHh0-unsplash_xs1n7p.jpg)
